client/quizGame.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+import { BrowserRouter, Link, Route, Routes } from "react-router-dom";
+import "./app.css";
+import { fetchJSON, postJSON } from "./http";
+import { useLoader } from "./useLoader";
+
+export function FrontPage() {
+  return (
+    <div className="main">
+      <h1>Quiz app</h1>
+      <div>
+        <Link to={"/question"}>
+          <button className="answerButton">Take a new quiz</button>
+        </Link>
+      </div>
+      <div>
+        <Link to={"/score"}>
+          <button className="answerButton">Show your score</button>
+        </Link>
+      </div>
+    </div>
+  );
+}
+
```

Comment:
Det kan være lurt å dele opp denne fila i flere filer. Det vil ikke være utslagsgivende for eksamen, men det er lurt å eksperimentere med hva dere liker